management/remote-wipe/main.py

```diff
@@ -0,0 +1,48 @@
+#!/bin/env python3
+
+import argparse
+import subprocess
+import json
+
+# TODO: Logging config
+
+
+def search_for_crypt(block_device):
+    if "children" in block_device:
+        for child in block_device["children"]:
+            crypt = search_for_crypt(block_device=child)
+            if crypt is not None:
+                return crypt
+    if "crypt" == block_device.get("type", ""):
+        return block_device["name"]
+
+    return None
+
+
+def wipe(saftey_on=True, cycles=10):
+    if saftey_on:
+        print("Safety on - not firing")
+        return
+
+    block_devices = json.loads(subprocess.check_output("lsblk --json".split(" ")))
+    crypts = []
+    for block_device in block_devices["blockdevices"]:
+        if crypt := search_for_crypt(block_device):
```

Comment:
I assume it's okay to be restricting this to `py>=3.8` through use of `:=`?

Reply:
its a good point - do we care about this being able to run on older distros? At least in its original use case - we don't

Reply:
I'm not particularly worried about it, and it's easy enough for a user to adapt so that it works.

Maybe just add a note or comment indicating that it requires 3.8 or higher?

Reply:
sure thing - that's done now